src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @include flex(center, '');
```

Comment:
The second parameter of the 'flex' mixin is an empty string. If this is intentional, consider removing it or replacing it with a more meaningful value to improve code readability.